extide/gradle/src/org/netbeans/modules/gradle/customizer/GradleExecutionPanel.java

```diff
@@ -69,17 +100,20 @@ public GradleExecutionPanel(Project project) {
     // <editor-fold defaultstate="collapsed" desc="Generated Code">//GEN-BEGIN:initComponents
     private void initComponents() {
 
-        cbTrustProject = new javax.swing.JCheckBox();
         lbTrustTerms = new javax.swing.JLabel();
         lbReadOnly = new javax.swing.JLabel();
-
-        org.openide.awt.Mnemonics.setLocalizedText(cbTrustProject, org.openide.util.NbBundle.getMessage(GradleExecutionPanel.class, "GradleExecutionPanel.cbTrustProject.text")); // NOI18N
+        cbTrustLevel = new javax.swing.JComboBox<>();
+        lbTrustLevel = new javax.swing.JLabel();
 
         lbTrustTerms.setVerticalAlignment(javax.swing.SwingConstants.TOP);
 
         lbReadOnly.setIcon(new javax.swing.ImageIcon(getClass().getResource("/org/netbeans/modules/gradle/resources/info.png"))); // NOI18N
         org.openide.awt.Mnemonics.setLocalizedText(lbReadOnly, org.openide.util.NbBundle.getMessage(GradleExecutionPanel.class, "GradleExecutionPanel.lbReadOnly.text")); // NOI18N
 
+        cbTrustLevel.setModel(new javax.swing.DefaultComboBoxModel<>(new String[] { "None", "Temporary", "Permanent" }));
```

Comment:
Not required here, just click on the use default. on the form editor to remove this line.

Reply:
Later on a new model can be created from Level.values()

Reply:
Fixed in 7e035b7